ivy/functional/frontends/torch/non_linear_activation_functions.py

```diff
@@ -16,3 +16,10 @@ def softmax(input, dim=None, dtype=None):
 
 
 softmax.unsupported_dtypes = ("float16",)
+
+
+def gelu(input, approximate='none'):
+    return ivy.gelu(input,approximate)
+
+
+gelu.unsupported_dtypes = ("float16",)
```

Comment:
Here, you should not have that extra ",“ there in this line.

Reply:
also, I check some other examples and I think this line should be : 
gelu.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
You should double-check if this is correct though.